packages/react/src/components/_CheckboxRadioTemplate/CheckboxRadioTemplate.tsx

```diff
@@ -69,13 +75,13 @@ export const CheckboxRadioTemplate = ({
         className,
       )}
       htmlFor={finalInputId}
-      isLabel={shouldHaveClickableLabel}
```

Comment:
I suggest making `&& !dontRenderLabelTag` a part of the computing of the `shouldHaveClickableLabel` variable instead.

Reply:
The problem with this is that `shouldHaveClickableLabel` is used on line 78 to determine whether to render a `label`. And at line 84 this value is negated, such that the code would always render a label, in one place or the other.